2017-2018/GameEngine/graphics/pacmanState_pb2.py

```diff
@@ -0,0 +1,292 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Is there something different about this file in the graphics package from the one in the comm package?

Reply:
Actually, I got an error trying to run the visualizer due to the duplication of the file. Apparently the library doesn't check for the duplicate name in some builds in some platforms, but it is supposedly incorrect behavior to do that.

Reply:
I realized I should have ran a diff, so I did, and it looks like the name of the Direction enum was outdated in one of them? In any case, it seems to run fine with both using the file in the `comm` package, so I'm getting rid of the duplicate in the graphics package.